qase-jest/examples/package.json

```diff
@@ -9,7 +9,9 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "jest": "^27.0.4",
+    "jest": "^27.0.4"
```

Comment:
Why jest is not a part of qase reporter?

Reply:
Because the user of our library has already written several tests and has jest as a dependency in his project. I'm not sure we should give him another version of jest in the dependency graph.